playbooks/group_vars/search.yml

```diff
@@ -0,0 +1,4 @@
+acs_play_search_artifact_name: alfresco-search-services
+acs_play_search_repository: "{{ nexus_repository.releases }}"
```

Comment:
I would not prefix these as they are not role arguments and are only used to build up the artifact url.

Reply:
using here the same prefix of a role prefix is asking for troubles